src/Sylius/Component/Resource/Exception/Repository/ExistingResourceException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Resource\Exception\Repository;
+
+/**
+ * @author Jan Góralski <jan.goralski@lakion.com>
+ */
+class ExistingResourceException extends \Exception
+{
+    public function __construct()
+    {
+        parent::__construct('Given resource already exists in the repository.');
+    }
+}
```

Comment:
Perhaps this namespace should be the other way around? I mean, we could put it in ``Sylius\Component\Resource\Repository\Exception\ExistingResourceException``? Otherwise it is a bit messy to look for appropriate exception in folder with all exception then "category" of the service which this exception is thrown by. What do you think?